src/index.test.tsx

```diff
@@ -199,4 +204,26 @@ describe("twc", () => {
     expect(title.classList.contains("font-medium")).toBe(true);
     expect(title.classList.contains("font-bold")).toBe(false);
   });
+
+  test("works with render props", () => {
+    const twx = createTwc({
+      compose: (fn, className) =>
+        typeof fn === "function"
+          ? composeRenderProps(className, (_prev, renderProps) =>
+              fn(renderProps),
+            )
```

Comment:
`_prev` should be composed with the return value of `fp(renderProps`. Wrap it with `twMerge` or `clsx` will suffice.

```javascript
...
        ? composeRenderProps(className, (prevClassName, renderProps) =>
                twMerge(prevClassName, fn(renderProps)),
            )
...
```